src/content/en/_shared/helpful.html

```diff
@@ -0,0 +1,36 @@
+{% framebox width="auto" height="auto" enable_widgets="true" %}
+<script>
+var label = window.parent.location.pathname,
```

Comment:
Per the Google JS Style guide, you should only have [one variable per declaration](https://google.github.io/styleguide/jsguide.html#features-one-variable-per-declaration) - will change and merge.